sig-architecture/charter.md

```diff
@@ -51,12 +51,6 @@ and opts-in to updates and modifications to [sig-governance].
 
 Federation of Subprojects as defined in [sig-governance]
 
-### To do
```

Comment:
Why are we removing these todo's when they have not yet been resolved?

Reply:
https://youtu.be/78EKp6fAsqE?t=1162